tests/test_remarshal.py

```diff
@@ -73,6 +78,14 @@ def tempFilename(self):
         self.temp_files.append(temp_filename)
         return temp_filename
 
+    def assertCborSame(self, output, reference):
+
+        # To date, Python’s CBOR libraries don’t support encoding to
```

Comment:
The Unicode single quote (’) in "don’t" prevents the tests from running in Python 2.

Reply:
Fixed.